src/core/template-helper.ts

```diff
@@ -14,57 +17,86 @@ interface IDxTemplate {
     render: (data: IDxTemplateData) => any;
 }
 
-interface ITemplateInfo {
+interface ITemplateBaseDto {
     name: string;
     prop: string;
     isNested: boolean;
     isComponent: boolean;
+    propsGetter: (prop: string) => any;
+}
+
+interface ITemplateDto extends ITemplateBaseDto {
```

Comment:
What about naming it `ITemplateDescriptor`? -`Dto` is from DB\ORM\etc world, personally I like `Descriptor` (or `Descr`) more